docs/GettingStarted/configuration/ExporterFailure.md

```diff
@@ -91,15 +91,17 @@ This is the list of options that can be applied to `env_from_secrets`, `env_from
 
 ###### API_USER
 
-- **Required:** yes
+- **Required:** no `jira`; yes for `servicenow`
     - Only applicable for [PROVIDER](#provider) set to `jira` or `servicenow`
+    - Required for the [PROVIDER](#provider) `servicenow`
```

Comment:
I felt this is a little confusing

I would not change anything in the current docs and just add a bullet in here with something like **"For [PROVIDER](#provider) set to `jira`, a Personal Access Token (PATs) can be used for [TOKEN](#token) without API_USER"**

Reply:
Closing, opened doc issue to make it consistent.